server/glsp-api/src/main/java/com/eclipsesource/glsp/api/di/GLSPModule.java

```diff
@@ -46,14 +46,16 @@ protected final void configure() {
 		bind(ModelElementOpenListener.class).to(bindModelElementOpenListener());
 		bind(ILayoutEngine.class).to(bindLayoutEngine());
 		bind(OperationConfiguration.class).to(bindOperationConfiguration());
-		bind(ModelTypeConfiguration.class).to(bindModelTypeConfiguration());
 		bind(ActionProvider.class).to(bindActionProvider());
 		bind(ActionHandlerProvider.class).to(bindActionHandlerProvider());
 		bind(OperationHandlerProvider.class).to(bindOperatioHandlerProvider());
 		bind(ServerCommandHandlerProvider.class).to(bindServerCommandHandlerProvider());
+		bind(ModelTypeConfigurationProvider.class).to(bindModelTypesConfigurationProvider());
 		configureMultibindings();
 	}
 
+
```

Comment:
Please remove those two blank lines. We should eventually add some save-actions that auto-format the files including removing more than two consecutive blank lines.